internal/services/videoanalyzer/resourceids.go

```diff
@@ -1,3 +1,4 @@
 package videoanalyzer
 
 //go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=VideoAnalyzer -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.Media/videoAnalyzers/analyzer1
+//go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=EdgeModules -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.Media/videoAnalyzers/analyzer1/edgeModules/edgemodule1
```

Comment:
this should be the singular here (since it's referring to a single item):

```suggestion
//go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=EdgeModule -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.Media/videoAnalyzers/analyzer1/edgeModules/edgemodule1
```